apps/annotation/mailgun.py

```diff
@@ -0,0 +1,20 @@
+import requests
+from django.conf import settings
+
+
+def send_mail(to_addr, subject, text, sender,
+              from_name='Przypis Powszechny',
+              to_name=None):
+    from_addr = '{}@{}'.format(sender, settings.PP_MAIL_DOMAIN)
+    response = requests.post(
+        settings.MAILGUN_API_URL,
+        auth=('api', settings.MAILGUN_API_KEY),
+        data={'from': '{} <{}>'.format(from_name, from_addr),
+              'to': '{} <{}>'.format(to_name, to_addr),
+              'subject': subject,
+              'text': text})
+
+    if not (200 <= response.status_code < 300):
+        raise RuntimeError('Request to {} unexpected status {}. Response: \n {}'.format(
```

Comment:
If we see it a kind of risky, let's just log it appropriately as `error` or even `critical`, It is a particular error then, not an unclassified generic 500 error.

Actually `request.post` is itself error-prone, what if there is typo in URL? Let's explicitly and verbosely handle those (known) cases. It might be here or in view using it, just in some way :)